tests/unit/mocks/service/documents.mock.js

```diff
@@ -9,6 +9,8 @@ jest.mock('@/service/documents', () => {
     waitDocumentVerified: jest.fn(),
     getDocumentById: jest.fn().mockResolvedValue(document),
     getRequiredDocumentsTypes: jest.fn().mockResolvedValue([]),
+    getSelectedDocuments: jest.fn().mockResolvedValue({}),
```

Comment:
Maybe some tests to be sure that these methods calling properly?

Reply:
add test for save selected documents